README.md

```diff
@@ -4001,6 +4001,37 @@ _You can enable the following settings in Xcode by running [this script](resourc
       private let step: Double.Stride
     }
     ```
+
+    Additioanlly, after grouping SwiftUI properties, its prefered that propeties are sorted by their dynamic property type. The order is determined by the first time a type appears:
```

Comment:
The discussion about how the format rule behaves seems more like an implementation detail rather than a "style rule". We should probably either clarify that this is describing the behavior of the formatter:

```suggestion
    Additionally, after grouping SwiftUI properties, its preferred that properties are grouped by their dynamic property type. The sort order applied by the formatter is determined by the first time a type appears:
```

or we could 

```suggestion
    Additionally, after grouping SwiftUI properties, its preferred that properties are grouped by their dynamic property type.
```

Saying "grouped by their dynamic property type" rather than "sorted by" seems better here, because "grouped" is the user-facing style requirement and "sorting" is kinda just the implementation detail of how we enforce the grouping.

(There are also a few typos in this sentence, corrected them in the two suggestions)